src/load/azext_load/commands.py

```diff
@@ -8,10 +8,9 @@
 # pylint: disable=too-many-lines
 # pylint: disable=too-many-statements
 
-# from azure.cli.core.commands import CliCommandType
-
 
 def load_command_table(self, _):  # pylint: disable=unused-argument
+    from azure.cli.core.commands import CliCommandType
```

Comment:
operations tmpl bit is missing I guess?

Reply:
There should be no change in this file as it is for CP. Will cleanup